src/state/api.js

```diff
@@ -54,3 +54,32 @@ export async function getProbeData(params, token) {
   });
   return data;
 }
+
+export async function getSearchResults(queryString, resultsLimit) {
```

Comment:
Since you're returning a `Promise` here (and the `await` keyword below is not necessary) we can just make this a regular function

```suggestion
export function getSearchResults(queryString, resultsLimit) {
```